cpp/include/cudf/detail/iterator.cuh

```diff
@@ -329,9 +329,12 @@ CUDF_HOST_DEVICE auto inline make_validity_iterator(column_device_view const& co
  *
  * For `p = *(iter + i)`, `p` is the validity of the scalar.
  *
+ * @tparam bool unused. This template parameter exists to enforce the same
+ *         template interface as @ref make_validity_iterator(column_device_view const&).
  * @param scalar_value The scalar to iterate
  * @return auto Iterator that returns scalar validity
  */
+template <bool safe = false>
```

Comment:
Is a uniform interface required for some calling code outside this PR? Introducing an unused template parameter for the sake of uniformity is an unclear tradeoff for me.

Reply:
It will be used in `lists::contains`. We want to call `get_validity_accessor` on an input that can be either a scalar or a column.

Reply:
Great! I am happy to see this is repeated elsewhere - I was not aware of that context. I'll resolve this.